docs/API.md

```diff
@@ -42,6 +42,19 @@ The data will automatically be sent to the user via Pusher.
 
 #### WRITE Response Errors
 When there is an error on a WRITE response (`jsonCode!==200`), the error must come back to the client on the HTTPS response. The error is only relevant to the client that made the request and it wouldn't make sense to send it out to all connected clients.
+
+Error messages should be returned and stored as a String under the `error` property. If absoultely needed, additional error properties can be stored under other, more specific fields that sit at the same level as `error`:
+```php
+[
+    'onyxMethod' => 'merge',
```

Comment:
Looks like we have a [dedicated constant](https://github.com/Expensify/Web-Expensify/blob/0384b1208459602ca88836b0b805a163dc9ee80c/lib/Onyx.php#L9) for this now:
```suggestion
    'onyxMethod' => Onyx::METHOD_MERGE,
```